src/components/RawQueryEditor.tsx

```diff
@@ -60,7 +60,7 @@ export const RawQueryEditor: React.FC<RawQueryEditorProps> = (props) => {
 
   const styles = useStyles2(getStyles);
 
-  const handleEditorMount = useCallback((editor: MonacoEditor, monaco: Monaco) => {
+  const handleEditorMount = useCallback(async (editor: MonacoEditor, monaco: Monaco) => {
```

Comment:
I don't think this `async` does anything if you don't have `await`

Reply:
right, I forgot to remove this.